glusterd2/commands/devices/adddevice.go

```diff
@@ -0,0 +1,71 @@
+package devicecommands
+
+import (
+	"net/http"
+
+	device "github.com/gluster/glusterd2/glusterd2/device"
```

Comment:
You don't need import alias here as it's used as is and there's no conflicting package name.